src/controller/python/test/test_scripts/base.py

```diff
@@ -560,9 +633,9 @@ async def TestMultiFabric(self, ip: str, setuppin: int, nodeid: int):
         self.logger.info("Shutdown completed, starting new controllers...")
 
         self.fabricAdmin = chip.FabricAdmin.FabricAdmin(vendorId=0XFFF1,
-                                                        fabricId=1, fabricIndex=1)
+                                                        fabricId=1, adminIndex=1)
         fabricAdmin2 = chip.FabricAdmin.FabricAdmin(vendorId=0xFFF1,
-                                                    fabricId=2, fabricIndex=2)
+                                                    fabricId=2, adminIndex=2)
```

Comment:
What about having multiple fabrics under adminIndex 1 (ExampleOperationalCredentialIssuer with mIndex == 1) ? This seems like we are still 1:1 Fabric->Root

Reply:
I think we'd still leave 'adminIndex' as 'the index associated with a specific fabric administrator instance', and create a separate 'CAIndex' that would refer to a new 'CredentialAuthority' object that would manage the example op creds issuer.